tests/integration/conftest.py

```diff
@@ -93,22 +110,77 @@ async def app(
     # mypy doesn't see that ActiveStatus has a name
     await model.wait_for_idle(status=ActiveStatus.name)  # type: ignore
 
+    # test that the application is online
+    for _ in range(10):
+        try:
+            response = requests.get(
+                f"http://{await get_mattermost_ip(ops_test, application)}:8065",
+                timeout=5
+            )
+            if response.status_code == 200:
+                break
+        except:
+            pass
+        # wait a bit
+        time.sleep(10)
+
     yield application
 
 
 @pytest_asyncio.fixture(scope="module")
-async def mattermost_ip(
+async def test_entities(
     ops_test: OpsTest,
     app: Application,
 ):
-    """Get the IP address of the first unit of mattermost.
+    mattermost_ip = await get_mattermost_ip(ops_test, app)
```

Comment:
Should we at least have some docstring here?

Reply:
I applied our docstring standards to the new utils module